hack/man-page-checker

```diff
@@ -19,7 +19,11 @@ cd $(dirname $0)/../docs/ || die "Please run me from top-level libpod dir"
 
 rc=0
 
-for md in *.1.md;do
+for md in *.1.md; do
```

Comment:
The podman version has a more complex way of skipping aliases, it patches a perl script, etc. Would rather use this way for now